src/config/ContractAddresses.ts

```diff
@@ -51,5 +40,6 @@ export const contractAddressesByChain: Record<number, ContractList> = {
     // sepolia
     [ContractTypes.AirSwapStaking_latest]:
       "0x8Bf384296A009723435aD5E8203DA5736b895038",
+    [ContractTypes.AirSwapToken]: "0x4092D6DBA9abB7450B9d91aA7ED2712935D63b39",
```

Comment:
Should not be an AST on BSC. Member dashboard only connects to Mainnet and Sepolia (testnet).